Lib/test/test_inspect.py

```diff
@@ -712,7 +712,8 @@ def assertArgSpecEquals(self, routine, args_e, varargs_e=None,
         self.assertEqual(varkw, varkw_e)
         self.assertEqual(defaults, defaults_e)
         if formatted is not None:
-            self.assertEqual(inspect.formatargspec(args, varargs, varkw, defaults),
+            with self.assertWarns(DeprecationWarning):
+                self.assertEqual(inspect.formatargspec(args, varargs, varkw, defaults),
```

Comment:
Please fix the indentation of the continuation line here and further in this file.